### PR TITLE
Able to edit and delete symptoms

### PR DESCRIPTION
**Pull request by xsaco07**
Now with long click event the user can delete and edit all information about the symptom

### CODE REVIEW

File: app/src/main/java/com/project/symptoms/activity/MainActivity.java [hunk 1]
```diff
@@ -65,7 +73,7 @@ protected void onCreate(Bundle savedInstanceState) {
     protected void onResume() {
         super.onResume();
         // This is needed to be done again because of the context used in DateTimeUtils
-        // changes when another view is also registered as picker
+        // changes when another view is also registered as piker
```
Comment (EmmanuelAB) [inline app/src/main/java/com/project/symptoms/activity/MainActivity.java]: why this change?

File: app/src/main/java/com/project/symptoms/view/BodyView.java [hunk 2]
```diff
@@ -198,12 +189,9 @@ private Rect calculateCenteredBounds() {
 
         int sideMargin = (int) (remainingFreeWidth / 2);
 
-        int left = sideMargin;
         int right = 3*sideMargin;
         int top = 0;
```
Comment (EmmanuelAB) [inline app/src/main/java/com/project/symptoms/view/BodyView.java:194]: I think that separating the four bounds variables from where they value come from is more readable (as it was before)

File: app/build.gradle [hunk 3]
```diff
@@ -16,6 +16,10 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+    compileOptions {
+        sourceCompatibility = 1.8
```
Comment (EmmanuelAB) [inline app/build.gradle:20]: what's this mean?

File: app/src/main/java/com/project/symptoms/view/BodyView.java (hunk 2 above)
Comment (xsaco07) [reply app/src/main/java/com/project/symptoms/view/BodyView.java:194]: The IDE it self showed a warning that is unnecessary to have that. But I can change it.

File: app/src/main/java/com/project/symptoms/activity/MainActivity.java (hunk 1 above)
Comment (xsaco07) [reply app/src/main/java/com/project/symptoms/activity/MainActivity.java]: Honestly, don't know why that is changed

File: app/build.gradle (hunk 3 above)
Comment (xsaco07) [reply app/build.gradle:20]: Needed to upgrade the Java version to be able to use lambda functionality which is supported from that version and higher.
